src/python/foglamp/core/api.py

```diff
@@ -6,25 +6,21 @@
 
 import time
 from aiohttp import web
-from foglamp import server
 
 __author__ = "Amarendra K. Sinha"
 __copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
 __license__ = "Apache 2.0"
 __version__ = "${VERSION}"
 
+__start_time = time.time()
+
+
 async def ping(request):
     """
     :return: basic health information json payload
     {'uptime': 32892} Time in seconds since FogLAMP started
     """
 
-    # Since foglamp can be started in foreground or as a daemon,
-    # need to check for both foglampd and foglamp
-    process_info = server.find_process_info('foglampd') or server.find_process_info('foglamp')
-
-    since_started = 0
-    if process_info is not None:
-        since_started = time.time() - process_info['start_time']
+    since_started = time.time() - __start_time
```

Comment:
we can keep this, but let's have real code to check named process from the system